src/wallet/rpcwallet.cpp

```diff
@@ -158,6 +158,30 @@ static std::string LabelFromValue(const UniValue& value)
     return label;
 }
 
+static void SetFeeEstimateMode(CWallet* const pwallet, CCoinControl& cc, const UniValue& estimate_mode, const UniValue& estimate_param)
```

Comment:
style-nit: pwallet can be a reference, since it is non-nullable?

Reply:
You mean `const CWallet& wallet` instead? I don't see that used very often anywhere, and it seems this thing is usually a `std::shared_ptr`.

Reply:
Heh, I'll just link to #15922